src/APIFunctions/User.js

```diff
@@ -151,3 +152,35 @@ export async function checkIfUserExists(email) {
   });
   return status;
 }
+
+/**
+ * This function takes in a list of current users and returns a
+ * filtered user list that is determined by the filter id.
+ * @param {array} users array of all registered users
+ * @param {integer} filterID represents what to filter email by
+ * @returns {array} filtered array of users
+ */
+export function filterUsers(users, filterID) {
+  let filteredUsers = users.filter((user) => {
+    const d = new Date();
+    if (filterID === userFilterType.VALID) {
+      if (
+        user.accessLevel >= membershipState.MEMBER
+        && user.membershipValidUntil > d.toISOString()
+      ) {
+        return user;
+      }
+    } else if (filterID === userFilterType.NON_VALID) {
+      if (
+        user.accessLevel === membershipState.NON_MEMBER ||
+        user.accessLevel === membershipState.PENDING
+      ) {
+        return user;
+      }
+    } else {
+      return user;
+    }
+    return '';
```

Comment:
you can return nothing and the `filter` function will omit that element if thats what you were trying to do
```suggestion
```